runtime/j9vm/j7vmi.c

```diff
@@ -380,6 +380,7 @@ java_lang_J9VMInternals_doPrivilegedWithException(JNIEnv* env)
 	return cached;
 }
 
+#if (JAVA_SPEC_VERSION == 8) && !defined(OPENJ9_BUILD) && (defined(WIN32) || defined (WIN64))
```

Comment:
Please remove the unnecessary test of `WIN64`.

The exports (for cmake and UMA), and `forwarders.m4` must be updated accordingly.

Reply:
It's not clear that there's an easy way to express `!defined(OPENJ9_BUILD)` for both cmake and UMA: Please just omit that condition, it won't make a significant difference for Semuru builds of Java 8.
```c
#if (JAVA_SPEC_VERSION == 8) && defined(WIN32)
```

Reply:
Why does WIN64 get excluded?

Reply:
`WIN64` is just redundant; it is not defined unless `WIN32` is also defined.